### PR TITLE
dumpling: hide password in dumpling log

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What is changed and how it works?
update dumpling and omit password

### Check List 

Tests 

 - Integration test

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -503,6 +503,8 @@ github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712 h1:R8gStypOBmpnHEx1q
 github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712/go.mod h1:PYMCGwN0JHjoqGr3HrZoD+b8Tgx8bKnArhSq8YVzUMc=
 github.com/pingcap/dumpling v0.0.0-20200819021059-26975af91c7b h1:RqxrVPu3e+nvwwwqhHedawNWkI/BQF4MgJRC+gcgnfk=
 github.com/pingcap/dumpling v0.0.0-20200819021059-26975af91c7b/go.mod h1:W1F7jfA4o29S7Mn8Gh9U81wvxdmEl72y5SQOqH0eDDg=
+github.com/pingcap/dumpling v0.0.0-20200821062544-fd4568290bec h1:11EBt45gLbPDwIApjTEdERBPFHKNbkwCZF9LpOOg2zU=
```
Comment (csuzhangxc) [inline go.sum:506]: please run `go mod tidy`, maybe we should add this to `Makefile` as we did for terror check.

Comment (GMHDBJD) [reply go.sum:504]: simply add a command in Makefile when build code. 7a72672

**Comment by lance6716** [comment]
LGTM

**Review by lance6716 — DISMISSED** [review]

**Comment by GMHDBJD** [comment]
@csuzhangxc add mod check in a119f49. PTAL again

**Review by csuzhangxc — DISMISSED** [review]
LGTM again

**Comment by GMHDBJD** [comment]
> in fact `PWD` is a bash built-in variable, holding the result of `pwd`. so no need to construct it

@lance6716 address in 6d46889. PTAL again~

**Pull request merged**